src/server/command/ssh.ts

```diff
@@ -3,10 +3,16 @@ import parseCommand from './parse';
 import logger from '../utils/logger';
 
 export default function sshOptions(
-  { pass, path, command, host, port, auth }: { [s: string]: string },
+  { pass, path, command, host, port, auth, knownhosts }: { [s: string]: string },
   key?: string
 ): string[] {
+  let hostChecking;
   const cmd = parseCommand(command, path);
+  if (knownhosts !== '/dev/null') {
```

Comment:
Can you switch this to a const using a ternary

Reply:
Fixed with https://github.com/butlerx/wetty/pull/250/commits/079f5ac3b3bfee334011c2b45c50c06b6ac227ac